src/Network/Network/AzureFirewall/NewAzureFirewallCommand.cs

```diff
@@ -154,6 +154,26 @@ public class NewAzureFirewallCommand : AzureFirewallBaseCmdlet
         )]
         public string[] PrivateRange { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Flag to indicate DNS Proxy is enabled or disabled"
+        )]
+        [ValidateSet("true", "false", IgnoreCase = false)]
+        public string DNSEnableProxy { get; set; }
```

Comment:
Please use switch parameters instead of "true"/"false" strings.
```suggestion
        public SwitchParameter EnableDNSProxy { get; set; }
```
If the default value is "true", then use `DisableDNSProxy` instead.

Reply:
Default is false. I changed this to be switch parameter.